ERCS/erc-7894.md

```diff
@@ -0,0 +1,143 @@
+---
+eip: 7894
+title: Social SBT
+description: Social SBT, A Soul-Bound Token for DAO Governance
```

Comment:
Your title and description need some work. "Social SBT" gives no useful information about the token, and your description is mostly repeating your title or expanding SBT. Try to make your title unique and meaningful, and your description should expand on your title instead of simply repeating it.